### DISCUSSION TITLE
Canvas renderer?

### DISCUSSION THREAD

**Discussion by mhkeller** [uwdata/mosaic#178] [Ideas]
Are there any plans to include a canvas renderer for vgplot? I could see it being useful if you're making multiple scatter plots and don't want to have to worry about the page getting slowed down by having a large amount of svg elements. 

I see that observablehq/plot has [some canvas support](https://observablehq.com/@fil/flight-paths) but I'm not an expert in that library so not sure if it's an API that vgplot can simply use or observablehq/plot would need to change.

**Comment by jheer** [comment]
No plans at the moment. First, Plot is designed to use SVG and we call it directly. The rendering is handled by that dependency. Second, we leverage SVG for interaction, for example by binding listeners to SVG elements generated by Plot. A switch to Canvas would break certain interactions given the current implementation.
